### DISCUSSION TITLE
Carousel on Mobile and Grid on Desktop

### DISCUSSION THREAD

**Discussion by fsa317** [saadeghi/daisyui#2774] [Q&A]
I am trying to show a "list" of cards.  On the desktop it makes sense as a grid of 4 columns.  However on mobile that would need to be 1 column and thus a lot of scrolling.  What I would prefer is a carousel on mobile.  I tried using a carousel on desktop but the experience stunk.  Anyone solved this type of dilemma?

**Comment by saadeghi** [comment]
You can use something like `carousel sm:grid sm:grid-cols-4` it will be carousel by default, but in screens larger than `sm` it will be a grid of 4 columns without horizontal scroll.

Let me know if you have a question.
